docs/source/install.rst

```diff
@@ -191,3 +191,22 @@ Run the configuration script with the extra flag:
     $ ../configure_cap -DENABLE_DOCUMENTATION=ON
 
 Open the file `index.html` in the directory `docs/html`.
+
+
+Install TPLs using spack
+------------------------
+
+Cap and its dependencies can be build using `spack <https://github.com/llnl/spack>`_. 
+After installing spack, you need to install the following packages:
+
+.. code::
+
+    $ spack install boost +graph +icu_support +mpi +python
+    $ spack install trilinos ~hdf5 ~hypre ~mumps +boost ^boost+graph+icu_support+mpi+python 
+    $ spack install dealii~arpack~gsl~hdf5~oce~petsc+trilinos+mpi \
+         ^trilinos~hdf5~hypre~mumps ^boost+graph+icu_support+mpi+python
+    $ spack install py-mpi4py
+    $ spack install py-matplotlib
```

Comment:
We don't use use ```h5py``` and spack does not have ```docopt```. I don't think that we use ```numpy``` either. Also spack will install the dependencies it needs. For examples, ```p4est``` is installed because ```deal.II``` is installed with ```mpi```.